proto/statesync/types.proto

```diff
@@ -13,6 +13,8 @@ message Message {
     ChunkResponse      chunk_response       = 4;
     LightBlockRequest  light_block_request  = 5;
     LightBlockResponse light_block_response = 6;
+    ParamsRequest      params_request       = 7;
+    ParamsResponse     params_response      = 8;
```

Comment:
I think this is fine for now, but I believe these messages and the channels should live in light client. This would allow the light client to be used in a general way within the p2p layer.

Reply:
talked with @cmwaters over slack about approaches for this. He mentioned this could be done in a non breaking way after 0.35. resolving this